colorer/src/pcolorer2/ChooseTypeMenu.cpp

```diff
@@ -127,8 +126,7 @@ void ChooseTypeMenu::MoveToFavorites(size_t index)
 size_t ChooseTypeMenu::AddFavorite(const FileType* fType)
 {
   size_t i;
-  for (i = favorite_idx; i < ItemCount && !(Item[i].Flags & MIF_SEPARATOR); i++)
```

Comment:
suggest to move out this 3 times used loop into separate function (LookupNextNonSeparator() ?)